server/utils/prisonerContexts.ts

```diff
@@ -0,0 +1,12 @@
+import type { PrisonerContext } from 'session'
+
+export default function getOrCreatePrisonerContext(
+  contexts: Record<string, PrisonerContext>,
+  prisonNumber: string,
+): PrisonerContext {
```

Comment:
It would be good to add this function directly into the context/session so that intellij suggests it rather than having to know it exists but I'm not sure of the best approach